sqlglot/dialects/presto.py

```diff
@@ -345,6 +348,17 @@ class Parser(parser.Parser):
         FUNCTION_PARSERS = parser.Parser.FUNCTION_PARSERS.copy()
         FUNCTION_PARSERS.pop("TRIM")
 
+        PROPERTY_PARSERS: t.Dict[str, t.Callable] = {
+            **parser.Parser.PROPERTY_PARSERS,
+            "SECURITY": lambda self: self._parse_security(),
+        }
+
```

Comment:
Let's move this to the base parser class

Reply:
But it is not supported by the generic sql syntax. It is supported by presto only, AFAIK. Are you sure? It breaks the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) recommendation

Reply:
I hear you, but this pattern is common in SQLGlot, so I'm suggesting this for consistency.

We do this when the logic doesn't clash with existing logic to make the base classes handle more SQL, which has two side-effects: allows for dialects to automatically inherit logic, usually making things work out-of-the-box, and making `parse_one` and `sql` more lenient, which is a nice QOL since you can play around without always setting a dialect.